File: src/main/java/pl/arturczopek/mycoach/model/add/NewProduct.java
package pl.arturczopek.mycoach.model.add;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

/**
 * @Author Artur Czopek
 * @Date 14-04-2017
 */

@Data
public class NewProduct {

    private long productId;

    @JsonProperty(required = true)
    private String productName;

    private byte[] image;
}

File: src/main/kotlin/pl/arturczopek/mycoach/model/add/NewProduct.kt
package pl.arturczopek.mycoach.model.add;

/**
 * @Author Artur Czopek
 * @Date 14-04-2017
 */

data class NewProduct(
        val productId: Long,
        val productName: String,
        val image: Array<Byte>?
)
